### ISSUE TITLE
Remove usage of deprecated `pkg_resources` module

### ISSUE THREAD

**Issue by rdimaio** [rucio/rucio#6975]
From https://setuptools.pypa.io/en/latest/pkg_resources.html:

> Use of pkg_resources is deprecated in favor of [importlib.resources](https://docs.python.org/3.11/library/importlib.resources.html#module-importlib.resources), [importlib.metadata](https://docs.python.org/3.11/library/importlib.metadata.html#module-importlib.metadata) and their backports ([importlib_resources](https://pypi.org/project/importlib_resources), [importlib_metadata](https://pypi.org/project/importlib_metadata)). Some useful APIs are also provided by [packaging](https://pypi.org/project/packaging) (e.g. requirements and version parsing). Users should refrain from new usage of pkg_resources and should work to port to importlib-based solutions.

We use `pkg_resources` here: https://github.com/rucio/rucio/blob/master/setuputil.py

**Comment by voetberg** [comment]
@rdimaio Is this issue still needed? Looks like pkg_resources is no longer in the setuputil

**Comment by rdimaio** [comment]
Good catch, was removed in https://github.com/rucio/rucio/commit/f0affaa7cb15e6cebe4d2cf25c0f579b868222b3

closing
